static/js/publisher/release/components/progressiveBar.js

```diff
@@ -171,7 +173,9 @@ class InteractiveProgressiveBar extends React.Component {
     };
 
     this.setState(newState);
-    onChange(target);
+    if (onChange) {
+      throttle(onChange(target), 250);
```

Comment:
`throttle` argument should be a function, does `onChange(target)` return a function to call?
Or should it be something like `throttle(() => onChange(target), 250`?

Reply:
You're right - and when it's called correctly it prevents the UI updating.